tools/crmadmin.c

```diff
@@ -408,6 +431,33 @@ admin_msg_callback(const char *buffer, ssize_t length, gpointer userdata)
             fprintf(stderr, "%s\n", state);
         }
 
+    } else if (DO_PACEMAKERD_HEALTH) {
+        xmlNode *data = get_message_xml(xml, F_CRM_DATA);
+        const char *state =
+            crm_element_value(data, XML_PING_ATTR_PACEMAKERDSTATE);
+        time_t pinged = (time_t) 0;
+        long long value_ll = 0;
+        crm_time_t *crm_when = crm_time_new(NULL);
```

Comment:
Not a big deal, but FYI, I recently added a crm_time_new_undefined() function for when you don't need it initialized. I needed it b/c I needed to know when it had been initialized; here it would just be a tiny bit more efficient.